packages/functions/src/emojiList.ts

```diff
@@ -0,0 +1,18 @@
+/**
+ * Generate a formatted list of items for Discord using emojis
+ * @param items - An array of strings to format
+ * @param doFirst - Whether the first line should have an list starting emoji (true) or be left blank to use as the title (false)
+ * @returns 
+ */
+export const emojiList = (items: string[], doFirst = false) => {
+	if (!items || items.length === 0) throw new Error("No items were specified")
```

Comment:
Typescript already checks if it is provided.
```suggestion
	if (items.length === 0) throw new Error("No items were specified")
```

If you wanted to be fancy you can use a tuple to make typescript throw the error if the array is empty using `items: [string, ...string[]]`